### PR TITLE
Write a simple program to periodically update validator set to the destination EVM blockchain

### LINKED ISSUE #1019: Write a simple program to periodically update validator set to the destination EVM blockchain
The program can be written as a separate go file and should poll for validator set from `26657/validators` and push the update to the destination smart contract. Should look like a real command line tool that can config poll interval and destination endpoint / contract.

### PR DESCRIPTION
**Pull request by Benzbeeb**
Fixed #1019

### CODE REVIEW

File: chain/cmd/bandevmbot/main.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	if err != nil {
+		panic(err)
+	}
+
+}
+
+func main() {
+
+	cmd := &cobra.Command{
+		Use:   "update-validator [rpc-uri] [contract-address] [node-uri] [priv-key] [gas-price] [poll-interval] ",
```
Comment (taobun) [inline chain/cmd/bandevmbot/main.go]: Change to flag arguments not positional arguments

File: chain/cmd/bandevmbot/main.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+		vals[idx] = ValidatorWithPower{
+			Addr:  crypto.PubkeyToAddress(*pubkey),
+			Power: big.NewInt(validator.VotingPower),
+		}
+
+	}
+	return vals
+}
+
+func update_validator(rpcURI string, address string, node string, privateKey string, gasPrice uint64) {
```
Comment (taobun) [inline chain/cmd/bandevmbot/main.go]: camelCase

**Review by taobun — CHANGES_REQUESTED** [review]

File: chain/cmd/bandevmbot/main.go [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+func updateValidators(rpcURI string, address string, node string, privateKey string, gasPrice uint64) {
+	vals := getValidators(node)
+	backgroundCtx := context.Background()
+	contractAddress := common.HexToAddress(address)
+	priv, err := crypto.HexToECDSA(privateKey)
+	botAddress := crypto.PubkeyToAddress(priv.PublicKey)
+	if err != nil {
+		panic(err)
+	}
+	evmClient, _ := ethclient.Dial(rpcURI)
```
Comment (sorawit) [inline chain/cmd/bandevmbot/main.go]: Why do we ignore the second return value?

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by sorawit — APPROVED** [review]
looks great

**Review by taobun — APPROVED** [review]

**Pull request merged**